README.md

```diff
@@ -84,7 +86,7 @@ Alternatively, you can download the lakeFS binaries and run them directly.
 Binaries are available at [https://github.com/treeverse/lakeFS/releases](https://github.com/treeverse/lakeFS/releases).
 
 
-#### Setting up a repository
+##### Setting up a repository
```

Comment:
This is not a subsection of "Download the binary", so I would prefer to avoid this change.